src/NuGet.Services.AzureSearch/Analysis/IdentifierCustomTokenFilter.cs

```diff
@@ -16,6 +16,7 @@ public static class IdentifierCustomTokenFilter
 
         public static WordDelimiterTokenFilter Instance = new WordDelimiterTokenFilter(
             Name,
+            catenateAll: true,
             splitOnCaseChange: true,
```

Comment:
Please update this class's comments and the functional tests to reflect this change.

Reply:
I didn't mean to add it just yet. removing it, will thoroughly test and add appropriate testing for this case.